schema/ont_papers.ttl

```diff
@@ -0,0 +1,68 @@
+@prefix : <http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix fabio: <http://purl.org/spar/fabio/> .
+@base <http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/> .
+
+<https://w3id.org/ocs/ont/papers/> rdf:type owl:Ontology ;
```

Comment:
Ontology IRI should not end with a / or # – only the prefix should. So, make it just `https://w3id.org/ocs/ont/papers`.